detection_rules/rule.py

```diff
@@ -1028,11 +1062,21 @@ def version_lock(self):
         # VersionLock
         from .version_lock import loaded_version_lock
 
+        err_msg = "Cannot access the version lock when the versioning strategy is configured to bypass the version" \
+                  " lock. Set `bypass_version_lock` to `false` in the rules config to use the version lock."
+
+        if RULES_CONFIG.bypass_version_lock is True:
+            raise ValueError(err_msg)
```

Comment:
need to adjust line breaking if moving

```suggestion

        if RULES_CONFIG.bypass_version_lock is True:
            err_msg = "Cannot access the version lock when the versioning strategy is configured to bypass the version" \
                  " lock. Set `bypass_version_lock` to `false` in the rules config to use the version lock."
            raise ValueError(err_msg)
```